README.md

```diff
@@ -26,26 +28,36 @@ github.com/heroku/go-getting-started
 $ heroku local
 ```
 
-Your app should now be running on [localhost:5000](http://localhost:5000/).
+Your app should now be running on [localhost:5006](http://localhost:5006/).
 
 ## Deploying to Heroku
 
 Using resources for this example app counts towards your usage. [Delete your app](https://devcenter.heroku.com/articles/heroku-cli-commands#heroku-apps-destroy) and [database](https://devcenter.heroku.com/articles/heroku-postgresql#removing-the-add-on) as soon as you are done experimenting to control costs.
 
+### Deploy on Heroku [Cedar](https://devcenter.heroku.com/articles/generations#cedar)
+
 By default, apps use Eco dynos if you are subscribed to Eco. Otherwise, it defaults to Basic dynos. The Eco dynos plan is shared across all Eco dynos in your account and is recommended if you plan on deploying many small apps to Heroku. Learn more about our low-cost plans [here](https://blog.heroku.com/new-low-cost-plans).
 
 Eligible students can apply for platform credits through our new [Heroku for GitHub Students program](https://blog.heroku.com/github-student-developer-program).
 
-```sh
+```text
```

Comment:
How do these codeblacks look with `term` rather than `text`?

Reply:
Looks like term is fine too https://gist.github.com/schneems/327af42109ce7aa5123bb3f05ba902ef